pkg/kubelet/kubelet_test.go

```diff
@@ -5020,3 +5025,14 @@ func retryWithExponentialBackOff(initialDuration time.Duration, fn wait.Conditio
 	}
 	return wait.ExponentialBackoff(backoff, fn)
 }
+
+func simulateVolumeInUseUpdate(
```

Comment:
I know this is a unittests, but this is a forever goroutine that can do no good. 
how about something like:
```go
func simulateVolumeInUseUpdate(volumeName api.Unique, stopCh struct {}) {
  for {
    select {
      case <- ticker.C:
        MarkVolumeAsReportedInUse()
      case <- stopCh: 
        return
    }
  }
}

stopCh := make(chan struct{})
defer close(stopCh)
go simuilatedVolumeInUse(stopCh)
```